HandheldCompanion/Controllers/NeptuneController.cs

```diff
@@ -87,21 +87,16 @@ public NeptuneController(PnPDetails details)
             RefreshControls();
 
             // Specific buttons
-            SupportedButtons.Add(ButtonFlags.L4);
-            SupportedButtons.Add(ButtonFlags.R4);
-            SupportedButtons.Add(ButtonFlags.L5);
-            SupportedButtons.Add(ButtonFlags.R5);
-
             SupportedAxis.Add(AxisLayoutFlags.LeftPad);
             SupportedAxis.Add(AxisLayoutFlags.RightPad);
 
+            SupportedButtons.AddRange(new List<ButtonFlags>() { ButtonFlags.OEM1, ButtonFlags.VolumeUp, ButtonFlags.VolumeDown });
+            SupportedButtons.AddRange(new List<ButtonFlags>() { ButtonFlags.L4, ButtonFlags.R4, ButtonFlags.L5, ButtonFlags.R5 });
             SupportedButtons.AddRange(new List<ButtonFlags>() { ButtonFlags.LeftPadClick, ButtonFlags.LeftPadTouch, ButtonFlags.LeftPadClickUp, ButtonFlags.LeftPadClickDown, ButtonFlags.LeftPadClickLeft, ButtonFlags.LeftPadClickRight });
             SupportedButtons.AddRange(new List<ButtonFlags>() { ButtonFlags.RightPadClick, ButtonFlags.RightPadTouch, ButtonFlags.RightPadClickUp, ButtonFlags.RightPadClickDown, ButtonFlags.RightPadClickLeft, ButtonFlags.RightPadClickRight });
 
             // Specific buttons that shouldn't be mappable
-            VirtualButtons.AddRange(new List<ButtonFlags>() { ButtonFlags.L4, ButtonFlags.R4, ButtonFlags.L5, ButtonFlags.R5 });
-            VirtualButtons.AddRange(new List<ButtonFlags>() { ButtonFlags.LeftPadClick, ButtonFlags.LeftPadTouch, ButtonFlags.LeftPadClickUp, ButtonFlags.LeftPadClickDown, ButtonFlags.LeftPadClickLeft, ButtonFlags.LeftPadClickRight });
-            VirtualButtons.AddRange(new List<ButtonFlags>() { ButtonFlags.RightPadClick, ButtonFlags.RightPadTouch, ButtonFlags.RightPadClickUp, ButtonFlags.RightPadClickDown, ButtonFlags.RightPadClickLeft, ButtonFlags.RightPadClickRight });
+            VirtualButtons = new(SupportedButtons);
```

Comment:
Why would you add all buttons to the virtual buttons list ?

Reply:
I'm not entirely sure what this list is for. I've checked references, but it seems the idea is that those keys are not target for the remapper.

In any case all the buttons were already on this list. I just added OEM and Vol keys there. So in general I did not change anything, just simplified the code.

What buttons should be on the VirtualButtons list by design?

Reply:
Ok, I know what you mean now.